rs/bitcoin/ckbtc/minter/src/state.rs

```diff
@@ -385,20 +385,20 @@ pub struct CkBtcMinterState {
 
     pub last_fee_per_vbyte: Vec<u64>,
 
-    /// The fee for a single KYT request.
-    pub kyt_fee: u64,
+    /// The fee for a single bitcoin check request.
+    pub check_fee: u64,
 
     /// The total amount of fees we owe to the KYT provider.
     pub owed_kyt_amount: BTreeMap<Principal, u64>,
```

Comment:
understanding question: We don't rename `owed_kyt_amount` because this is no longer used, but still needed to replay events, correct?